docs/source/operations/operations_computation.rst

```diff
@@ -0,0 +1,741 @@
+a height. Of course the results of such an operation are questionable, hence the
+ballpark qualifier and a unknown accuracy advertized for such an operation.
+
+Vertical CRS to a Vertical CRS
+---------------------------------------------------------------------------------
+
+Overall logic is similar to the above case. There might be direct operations in
+the PROJ database, involving grid transformations or simple offsets. The fallback
+case is to synthetize a ballpark transformation.
+
+This is implemented by the ``createOperationsVertToVert`` method
+
+.. code-block:: shell
+
+    $ projinfo -s "NGVD29 depth (ftUS)" -t "NAVD88 height" --spatial-test intersects -o PROJ
+
+    Candidate operations found: 3
+    -------------------------------------
+    Operation n°1:
+
+    unknown id, Inverse of NGVD29 height (ftUS) to NGVD29 depth (ftUS) + NGVD29 height (ftUS) to NGVD29 height (m) + NGVD29 height (m) to NAVD88 height (3), 0.02 m, USA - CONUS east of 89°W - onshore
+
+    PROJ string:
+    +proj=pipeline +step +proj=axisswap +order=1,2,-3 +step +proj=unitconvert +z_in=us-ft +z_out=m +step +proj=vgridshift +grids=vertcone.gtx +multiplier=0.001
+
+    -------------------------------------
+    Operation n°2:
+
+    unknown id, Inverse of NGVD29 height (ftUS) to NGVD29 depth (ftUS) + NGVD29 height (ftUS) to NGVD29 height (m) + NGVD29 height (m) to NAVD88 height (2), 0.02 m, USA - CONUS 89°W-107°W - onshore
+
+    PROJ string:
+    +proj=pipeline +step +proj=axisswap +order=1,2,-3 +step +proj=unitconvert +z_in=us-ft +z_out=m +step +proj=vgridshift +grids=vertconc.gtx +multiplier=0.001
+
+    -------------------------------------
+    Operation n°3:
+
+    unknown id, Inverse of NGVD29 height (ftUS) to NGVD29 depth (ftUS) + NGVD29 height (ftUS) to NGVD29 height (m) + NGVD29 height (m) to NAVD88 height (1), 0.02 m, USA - CONUS west of 107°W - onshore
+
+    PROJ string:
+    +proj=pipeline +step +proj=axisswap +order=1,2,-3 +step +proj=unitconvert +z_in=us-ft +z_out=m +step +proj=vgridshift +grids=vertconw.gtx +multiplier=0.001
+
+
+Compound CRS to a Geographic CRS
+---------------------------------------------------------------------------------
+
+A typical example of a Compound CRS is a CRS made of a geographic or projected CRS
+as the horizontal component, and a vertical CRS. E.g. "NAD83 + NAVD88 height"
+
+When the horizontal component of the compound source CRS is a projected CRS, we
+first look for the operation from this source CRS to another compound CRS made
+of the geographic CRS base of the projected CRS,
+like "NAD83 / California zone 1 (ftUS) + NAVD88 height" to "NAD83 + NAVD88 height",
+which ultimately goes to one of the above described case. Then we can consider
+the transformation from a compound CRS made of a geographic CRS to another geographic CRS.
+
+It first starts by the vertical transformations from the vertical CRS of the
+source compound CRS to the target geographic CRS, using the strategy detailed
+in verttogeog_
```

Comment:
```suggestion
in :ref:`verttogeog_`
```

I guess?

Reply:
Both are possible I think (see http://docutils.sourceforge.net/docs/user/rst/quickref.html#internal-hyperlink-targets). I'm always confused with rst vs sphinx linking mechanisms. At least, the syntax I used works

Reply:
except that the title wasn't expanded. So fixing that